Gemfile

```diff
@@ -0,0 +1,25 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.3.4'
+
+gem 'rails', '~> 5.2.3'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+# gem 'jbuilder', '~> 2.5'
```

Comment:
why leave these out commented gems in here?

Reply:
fixed

Reply:
we only left `cors`. we will `bundle` it with the next one.